### ISSUE TITLE
Entity SQL Queries

### ISSUE THREAD

**Issue by AceCoderLaura** [dotnet/efcore#19006]
In EF6 you could fetch records from a database with Entity SQL convertible from a string. Will/does this exist in EF Core?
```
//Querying with Object Services and Entity SQL
string sqlString = "SELECT VALUE st FROM SchoolDBEntities.Students " +
                    "AS st WHERE st.StudentName == 'Bill'";
    
var objctx = (ctx as IObjectContextAdapter).ObjectContext;
                
ObjectQuery<Student> student = objctx.CreateQuery<Student>(sqlString);
Student newStudent = student.First<Student>();
```
([Code from entityframeworktutorial.net](https://www.entityframeworktutorial.net/Querying-with-EDM.aspx))

**Comment by ajcvickers** [comment]
@AceCoderLaura We have no plans to bring Entity SQL or anything like it to EF Core. Regular SQL is supported using the `FromSql` methods.

**Comment by AceCoderLaura** [comment]
But isn't regular SQL constained to only work with one database platform?

**Comment by ErikEJ** [comment]
With EF Core you can use LINQ or platform specific SQL statements

**Comment by ajcvickers** [comment]
@AceCoderLaura Yes. However, the use of ESQL was low with EF6 and requests for it in EF Core have been minimal. Given the implementation cost is very high its not something we plan to invest in.
